packages/embark-ui/src/components/Communication.js

```diff
@@ -37,6 +37,13 @@ class Communication extends Component {
     }
   }
 
+  handleEnter(e, cb) {
+    if (e.which === 13) {
```

Comment:
FYI, you can also do `e.key === 'Enter'`

Reply:
I agree that `e.key === 'Enter'` is preferable.

Some helpful info here: https://medium.com/@uistephen/keyboardevent-key-for-cross-browser-key-press-check-61dbad0a067a